package.json

```diff
@@ -808,7 +808,9 @@
         "@azure/arm-resources": "^4.0.0",
         "@azure/ms-rest-js": "^2.2.1",
         "dotenv": "^6.2.0",
+        "fast-xml-parser": "^3.19.0",
```

Comment:
Please see if you can avoid adding dependencies for your PR, as each dependency added can negatively affect activation performance. I'm also asking this because I believe the dependencies are only used once and Java is not a very high usage scenario for us

Reply:
>  as each dependency added can negatively affect activation performance

the size of `fast-xml-parser` is small, it doesn't even have any dependency for production(only devDependencies)
https://www.npmjs.com/package/fast-xml-parser
as far as I know, only the first level functions/classes will be loaded (but none of them will be called) on `import`/`require`.
if we implement the same logic by ourselves, we also need to make some functions/classes, they also need to be loaded first. (the only overhead of using external module is it may load some/a few number of un-required methods for us) I don't think we need to consider this performance degradation.

> I'm also asking this because I believe the dependencies are only used once

`only used once` is not a good reason for us to reinvent the wheel, 
and the logic of converting xml into json is actually not that simple.

> Java is not a very high usage scenario for us

### **Java is First Class Citizen on Azure**
**`low usage` is the start point and a good reason for us to improve the UX for Java developers but not to give up them.**
Java developers matter for Microsoft and for Azure especially when **the number for java vscode developers increase rapidly.**